apps/schedule/feeds.py

```diff
@@ -32,26 +35,150 @@ def _format_event_description(event):
 
 
 @schedule.route("/schedule/<int:year>.json")
-@cross_origin(methods=["GET"])
 def schedule_json(year):
     if year != event_year():
         return feed_historic(year, "json")
 
-    if not feature_enabled('SCHEDULE'):
+    if not feature_enabled("SCHEDULE"):
         abort(404)
 
-    schedule = [_convert_time_to_str(p) for p in _get_scheduled_proposals(request.args)]
+    if request.args.get('format') == 'array':
```

Comment:
Also feels wrong to do it like this. I don't know. Feedback please! (I'm happy to expose the schema-compatible json on another endpoint, i just have no clue what would be a good name)

Reply:
I think having this exposed at /schedule-c3voc.json would be preferable - it's then clear what format it is, and it's better to return different formats based on path rather than a querystring parameter.

Reply:
i changed it to `/schedule/frab-<year>.json` because that's where the json export originates from.